actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -172,11 +172,7 @@ pub fn call_generic<RT: Runtime>(
 
         let dst: EthAddress = dst.into();
         if is_reserved_precompile_address(&dst) {
-            let context = PrecompileContext {
-                call_type: kind,
-                gas_limit: effective_gas_limit(system, gas),
```

Comment:
This would unconditionally query the remaining gas available, even if we don't need it. In practice, the only precompile that needs it is the "native call" precompile.